### PR TITLE
Homepage layout

### PR DESCRIPTION
**Pull request by vstanyshevskyy**
Implements #15 
screenshot-2018-6-19 - - 1
screenshot-2018-6-19 - -

### CODE REVIEW

File: src/components/Feedback/Feedback.js [hunk 1]
```diff
@@ -1,162 +1,15 @@
...
-}
+export default props => (
+  <form
+    action={`https://formspree.io/${props.email}`}
+    method="POST"
+  >
+    <label htmlFor="form-message" >Повідомлення:</label>
+    <textarea id="form-message" name="message" required placeholder="Повідомлення" />
+    <label htmlFor="form-contact">Email, щоб ти точно отримав відповідь</label>
+    <input id="form-contact" type="text" name="_replyto" placeholder="example@test.com" required />
```
Comment (zoryana94) [inline src/components/Feedback/Feedback.js]: It's not necessary to set the type="text" directly... you can omit it

File: src/components/articles-cards/index.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Link from 'gatsby-link';
+import moment from 'moment';
+import './index.less';
+
+export default props => (
+  <aside className={`articles-cards-list ${props.useVerticalLayout ? 'articles-cards-list--vertical' : ''}`}>
```
Comment (zoryana94) [inline src/components/articles-cards/index.js]: Please, use classnames here
```
import classNames from 'classnames';

const articlesCardsListClass = classNames('articles-cards-list', {
    'articles-cards-list--vertical': props.useVerticalLayout
});

<aside className={articlesCardsListClass}...>
```

File: src/components/articles-cards/index.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+          props.titleLink ?
+            <Link to={props.titleLink}>{props.titleText}</Link> :
+            props.titleText
+        }
+      </span>
+    </h2>
+    <div className="collection">
+      <div className="collection-viewport">
+        <ul className="category-items">
+          { (props.articles || []).map((article, index) => (
```
Comment (zoryana94) [inline src/components/articles-cards/index.js]: I believe it's better to use destructuring... you can pass the [] by default

File: src/components/articles-cards/index.less [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    margin-bottom: 30px;
+    article {
+      overflow: hidden;
+    }
+  }
+  .item-featured-media {
+    float: left;
+    width: 50%;
+    max-width: 364px;
+    margin: 0 20px 0 0;
```
Comment (zoryana94) [inline src/components/articles-cards/index.less]: Please, add the empty lines between the style definitions

File: src/pages/home-page.less [hunk 5]
```diff
@@ -1,47 +1,238 @@
...
+      top: 0;
+      text-align: center;
+      background-color: #f2f2f2;
+    }
+  }
+  .article-container {
+    padding: 3px;
+  }
+  article {
+
```
Comment (zoryana94) [inline src/pages/home-page.less]: Remove the redundant empty line, please

File: src/pages/home-page.less [hunk 6]
```diff
@@ -1,47 +1,238 @@
...
-    color: #000;
+    background-position: center center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    position: absolute;
+    left: 0;
+    right: 0;
+    top: 0;
+    bottom: 0;
+    a {
```
Comment (zoryana94) [inline src/pages/home-page.less]: It's better to use the classes

File: src/pages/index.js [hunk 7]
```diff
@@ -1,75 +1,121 @@
...
-            <Articles articles={articles} />
-          </div>
-          <Feedback />
-        </div>
+      <div className="container-fluid latest-articles-container">
+        <Row>
+          <h2 className="latest-articles-title">
+            <strong>{homepageSettings.latestArticlesHeader}</strong>
+          </h2>
+          { latestArticles.map((article, index) => {
```
Comment (zoryana94) [inline src/pages/index.js]: Please, move it to the separate function

File: src/components/Feedback/Feedback.js (hunk 1 above)
Comment (vstanyshevskyy) [reply src/components/Feedback/Feedback.js]: True

File: src/components/articles-cards/index.js (hunk 2 above)
Comment (vstanyshevskyy) [reply src/components/articles-cards/index.js]: Nice, was not aware of it

File: src/pages/home-page.less (hunk 5 above)
Comment (vstanyshevskyy) [reply src/pages/home-page.less]: ok

File: src/components/articles-cards/index.less (hunk 4 above)
Comment (vstanyshevskyy) [reply src/components/articles-cards/index.less]: I don't want to take care of empty line. Will log an issue to integrate some LESS linter

**Pull request merged**